Projects/UOContent/Items/Containers/Fillable Containers/FillableContainer.cs

```diff
@@ -19,7 +19,11 @@ private void DeserializeRespawnTimer(TimeSpan delay)
         }
     }
 
-    public FillableContainer(int itemID) : base(itemID) => Movable = false;
+    public FillableContainer(int itemID) : base(itemID)
+    {
+        Movable = false;
+        ContentType = FillableContentType.None;
```

Comment:
I approved but I may change this to the field instead of the property to avoid the overhead setter.